### PR TITLE
Fix some recipe welder charges

### PR DESCRIPTION
**Pull request by Cyrano7**

### CODE REVIEW

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -2201,7 +2201,7 @@
   "reversible": true,
   "autolearn": false,
   "qualities" : [ {"id":"SAW_M","level":1,"amount":1} ],
-  "tools": [ [ [ "welder", 500 ], [ "toolset", 750 ] ] ],
+  "tools": [ [ [ "welder", 500 ], [ "toolset", 750 ], [ "welder_crude", 750 ] ] ],
```
Comment (illi-kun) [inline data/json/recipes/recipe_deconstruction.json]: I'm not able to use fully charged makeshift welder (450) with extra batteries in my inventory for crafting a drum using this receipt.

Comment (Coolthulhu) [inline data/json/recipes/recipe_deconstruction.json]: It's a deconstruction recipe.
Welder data is more of a convention here - it wouldn't be used anyway, as welder tool requirement is changed into metal sawing requirement on deconstruction.

Comment (illi-kun) [inline data/json/recipes/recipe_deconstruction.json]: @Coolthulhu I've totally missed that point, thanks for the note.

**Comment by pisskop** [comment]
why is the crude welder 66% more expensive in vehicle cobstruction but %50 in recipes?

**Comment by illi-kun** [comment]
@pisskop it's better to this raise this question in #17206.

**Pull request merged**